contracts/Nexus.sol

```diff
@@ -220,7 +216,9 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
         _initModuleManager();
         (address bootstrap, bytes memory bootstrapCall) = abi.decode(initData, (address, bytes));
         (bool success, ) = bootstrap.delegatecall(bootstrapCall);
+        
         require(success, NexusInitializationFailed());
+        require(_hasValidators(), MissingValidator());
```

Comment:
this function checks at least one validator is there?

Reply:
yes